webapp/src/main/java/com/box/l10n/mojito/quartz/QuartzConfig.java

```diff
@@ -32,6 +33,9 @@ public class QuartzConfig {
   @Autowired(required = false)
   List<JobDetail> jobDetails = new ArrayList<>();
 
+  @Value("${l10n.org.quartz.scheduler.enabled:true}")
```

Comment:
we probably want to move this to QuartzPropertiesConfig to have all configurations in one place, and later support the multiple scheduler setup. Those config classes may need some deeper refactoring too.